packages/components/atoms/f-form-field/src/components/_tests/FormField.test.js

```diff
@@ -993,45 +993,17 @@ describe('FormField', () => {
                 });
             });
         });
-
-        describe('`generateUniqueId ::`', () => {
-            let wrapper;
-
-            beforeEach(() => {
-                wrapper = shallowMount(FormField);
-            });
-
-            it('should exist', () => {
-                // Act & Assert
-                expect(wrapper.vm.generateUniqueId).toBeDefined();
-            });
-
-            describe('when invoked', () => {
-                it('should make a call to uuid to generate a unique ID', () => {
-                    // Arrange
-                    const uuidSpy = jest.spyOn(uuid, 'v4');
-
-                    // Act
-                    wrapper.vm.generateUniqueId();
-
-                    // Assert
-                    expect(uuidSpy).toHaveBeenCalled();
-                });
-            });
-        });
     });
 
     describe('created ::', () => {
         it('should invoke `generateUniqueId` so unique IDs within the form match when the component renders serverside', () => {
```

Comment:
Nit: `generateUniqueId` doesn't exist anymore, and I think I'd prefer this test to check that `uniqueIdentifier` isn't still null or an empty string, etc.

Reply:
🦅